Error: Cannot find module './syntax-css-in-js' [v0.87.2]

<!-- Please answer the following. We close issues that don't. -->

> Clearly describe the bug

Stylelint no longer works and throws an error on every keystroke

> What code is needed to reproduce this issue?

None, editing an average JS file


> What vscode-stylelint configuration is needed to reproduce the bug?

None, editing an average JS file


> Is this issue related to autofix? (`editor.codeActionsOnSave`)

Probably not, error throws without needing to save

> Which version of vscode-stylelint are you using?
```
v0.87.2
```

> Which version of stylelint are you using?

Not sure?

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

stylelint to work and not throw an error

> What actually happened (e.g. what warnings or errors you are getting)?

Error: Cannot find module './syntax-css-in-js'

`Error: Cannot find module './syntax-css-in-js' Require stack: - /Users/jwm/.vscode/extensions/stylelint.vscode-stylelint-0.87.2/dist/server.js at Function.Module._resolveFilename (internal/modules/cjs/loader.js:934:15) at Module._load (internal/modules/cjs/loader.js:779:27) at Function.f._load (electron/js2c/asar_bundle.js:5:12913) at Module.require (internal/modules/cjs/loader.js:1006:19) at require (internal/modules/cjs/helpers.js:88:18) at dD (/Users/jwm/.vscode/extensions/stylelint.vscode-stylelint-0.87.2/dist/server.js:57:49885) at Object.get (/Users/jwm/.vscode/extensions/stylelint.vscode-stylelint-0.87.2/dist/server.js:57:49933) at BNt (/Users/jwm/.vscode/extensions/stylelint.vscode-stylelint-0.87.2/dist/server.js:232:17424) at Object.pNt (/Users/jwm/.vscode/extensions/stylelint.vscode-stylelint-0.87.2/dist/server.js:232:19679) at new e (/Users/jwm/.vscode/extensions/stylelint.vscode-stylelint-0.87.2/dist/server.js:99:1222)
`

<!-- If the bug can be reproduced using the stylelint CLI, e.g. `npx stylelint "src/**/*.css"`, please create the issue in the stylelint issue tracker (https://github.com/stylelint/stylelint/issues/new?template=REPORT_A_BUG.md) instead. -->

--- a/bundle.js
+++ b/bundle.js
@@ -1,0 +1,45 @@
+'use strict';
+
+/* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable no-process-exit */
+/* eslint-disable node/no-unpublished-require */
+
+const fs = require('fs-extra');
+const esbuild = require('esbuild');
+const glob = require('fast-glob');
+
+const args = new Set(process.argv.slice(2));
+
+async function bundle() {
+	const entryPoints = [
+		'index.js',
+		'server.js',
+		// Necessary because bundler cannot recognize lazy loads in
+		// https://github.com/stylelint/stylelint/blob/13.13.1/lib/syntaxes/index.js#L9-L15
+		...(await glob('node_modules/stylelint/lib/syntaxes/syntax-*.js')),
+	];
+
+	for (const path of await glob('dist/*')) {
+		await fs.remove(path);
+	}
+
+	try {
+		await esbuild.build({
+			entryPoints,
+			entryNames: '[name]',
+			bundle: true,
+			outdir: 'dist',
+			external: ['vscode'],
+			format: 'cjs',
+			platform: 'node',
+			logLevel: 'info',
+			watch: args.has('--watch'),
+			sourcemap: args.has('--sourcemap'),
+			minify: args.has('--minify'),
+		});
+	} catch {
+		process.exit(1);
+	}
+}
+
+bundle();

--- a/package.json
+++ b/package.json
@@ -233,7 +233,7 @@
   },
   "scripts": {
     "ci:download-vscode": "node ci-download-vscode.js",
-    "bundle-base": "esbuild ./index.js ./server.js --bundle --outdir=dist --external:vscode --format=cjs --platform=node",
+    "bundle-base": "node bundle.js",
     "bundle": "npm run bundle-base -- --sourcemap",
     "bundle-watch": "npm run bundle-base -- --sourcemap --watch",
     "format": "prettier . --write",

